.github/workflows/dev-tool-test.yml

```diff
@@ -34,3 +34,10 @@ jobs:
     with:
       command: cd ./tools/web3js-example/ && npm run test
       directory: ./tools/web3js-example
+
+  golang:
+    name: Golang
+    uses: ./.github/workflows/dev-tool-workflow.yml
+    with:
+      command: cd ./tools/golang-json-rpc-tests/ && go run . && go run . --wss
```

Comment:
Should the command be broken up to allow for the `wss` flow to run after the http flow - even if the http flow fails

Reply:
@Nana-EC Good point. Done.